hooks/utils.ts

```diff
@@ -0,0 +1,6 @@
+export const capitaliseWords = (str: string) => {
```

Comment:
This is fine. In fact, I like that you just created a util function. But maybe think about https://www.npmjs.com/package/inflection